docs/docs/page-query.md

```diff
@@ -54,7 +54,9 @@ const HomePage = () => {
 }
 ```
 
-Below our `HomePage` component declaration, export a new constant called `query`, and set its value to be a `graphql` [tagged template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) with the query between two backticks:
+Below our `HomePage` component declaration, export a new constant called `query`. The name of the constant isn't important, Gatsby just looks for an exported `graphql` string from the file. Note that you can only have one page query per file.
```

Comment:
some how i am confused with the wording `page query` there - it is never used before in this doc...

PS: only the file name of the MD is `page-query.md`

Reply:
i see it's later also used, maybe we introduce a small definition at the beginning. for example distinction between a `normal` query and then that is there a `page query` and thats why this document explains it ...

Reply:
Taking the opportunity to differentiate between types of queries is always a good idea! This one should definitely focus on page query since that's what the doc is about, so any definitions should be brief.